tools/cp-caps/README.md

```diff
@@ -123,3 +124,13 @@ diag gpio get 2 ------------------------------------------ NotSupported
 diag gpio set 2 0 ---------------------------------------- NotSupported
 diag gpio set 2 1 ---------------------------------------- NotSupported
 ```
+
+### Test Data Poll
+
+The parameter `-p` or `--data-poll` starts to test whether the RCP supports data poll.
+
+```bash
+$ DUT_ADB_USB=1269UCKFZTAM95OR REF_CLI_SERIAL=/dev/ttyACM0 python3 ./tools/cp-caps/rcp_caps_test.py -p
+Data Poll Receiver --------------------------------------- OK
```

Comment:
I think the name is not consice because the child sends the data poll and the parent receives it. But here 'Data Poll Receiver' is actually the SED. So I suggest we use 'Data Poll Child' and 'Data Poll Parent' in the doc and following tests.